engines/grim/grim.cpp

```diff
@@ -274,6 +274,12 @@ GfxBase *GrimEngine::createRenderer(int screenW, int screenH) {
 #endif
 			0;
 
+	// For Grim Fandango, Korean fan translation can only use OpenGL renderer
```

Comment:
Could you please explain this restriction?

Reply:
The Korean fan translation uses TTF fonts and doesn't support openGL shaders (we haven't implemented them), so only use OpenGL.